packages/x-date-pickers/src/PickersLayout/PickersLayout.types.ts

```diff
@@ -38,16 +38,9 @@ export interface ExportedPickersLayoutSlots<
   >;
 }
 
-interface PickersLayoutActionBarOwnerState<
-  TValue,
-  TDate extends PickerValidDate,
-  TView extends DateOrTimeViewWithMeridiem,
-> extends PickersLayoutProps<TValue, TDate, TView> {
-  wrapperVariant: WrapperVariant;
-}
-
-interface PickersShortcutsOwnerState<TValue> extends PickersShortcutsProps<TValue> {
+export interface PickersLayoutOwnerState<TValue> extends PickerOwnerState<TValue> {
   wrapperVariant: WrapperVariant;
+  isLandscape: boolean;
```

Comment:
We could move those two variables to `PickerOwnerState` (they are available at the `usePicker` level).

But not sure how much `isLandscape` makes sense to be accessible everywhere.